velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -85,6 +87,56 @@ class E2EFilterTest : public E2EFilterTestBase {
     return std::make_unique<ParquetReader>(std::move(input), opts);
   }
 
+  void testLocalFileSinkCloseAbort(bool isAbort) {
+    auto rowType = ROW({INTEGER()});
+    std::vector<RowVectorPtr> batches;
+    batches.push_back(std::static_pointer_cast<RowVector>(
+        test::BatchMaker::createBatch(rowType, 2, *leafPool_, nullptr, 0)));
+    LocalFileSink::registerFactory();
+    auto root = exec::test::TempDirectoryPath::create();
+    auto filePath = fs::path(root->path) / "xxx/yyy/zzz/test_file.ext";
+
+    ASSERT_FALSE(fs::exists(filePath.string()));
```

Comment:
Not required as we are using a random directory.

Reply:
Fixed